.github/workflows/presubmit.yml

```diff
@@ -23,3 +23,7 @@ jobs:
     - name: Run presubmit checks
       run: |
         FUZZBENCH_TEST_INTEGRATION=1 make presubmit
+
+    - name: Run end to end CI test
+      run: |
+        make run-ci-test
```

Comment:
I don't really think run-ci-test is descriptive of what is happening. Can we name it end-to-end-test?

Reply:
Done